### ISSUE TITLE
Add support for resourceTags in `copilot env deploy`

### ISSUE THREAD

**Issue by efekarakus** [aws/copilot-cli#3874]
Is it feasible to add support for `resourceTags` for `copilot env deploy` command at https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/env_deploy.go#L29? This thought is similar to what is currently available for `copilot svc deploy` command/s at https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/svc_deploy.go#L40.

In general, it could be beneficial to have `resourceTags` option on multiple cli commands(where feasible) that may allow users to apply custom tags on all the resources which copilot creates or adding tags on resources already created by copilot. Having custom tags is generally a hard requirement at many organizations where users may be expected to tag all the resources. Thoughts?</div>

### Discussion
Discussed in https://github.com/aws/copilot-cli/discussions/3872

**Comment by h5aaimtron** [comment]
@efekarakus I think it would be useful to add custom resource tags for everything from within the manifest as well if possible.

**Comment by efekarakus** [comment]
@h5aaimtron 👍 would you expect the `tags` in the environment manifest to apply to all resources within an environment? More specifically for example, would you expect a Load Balanced Web Service to "inherit" the tags in the environment manifest?

**Comment by h5aaimtron** [comment]
@efekarakus yeah, I guess I would or at least provide a flag to say so. I believe that is how it is working in our CDK stacks.

**Comment by proof-nicholas** [comment]
Any ETA on this?

**Comment by harvindkumark** [comment]
@efekarakus  Is it possible to add tags at the environment level. It would be really helpful if you could update us on this. Thanks.
